src/main/java/com/hello/hello/Student.java

```diff
@@ -0,0 +1,42 @@
+package com.hello.hello;
+
+public class Student {
```

Comment:
@simamumu89 
患者は`patient` なのかなと思ったのですが、いかがでしょうか！？
```suggestion
public class Patient {
```

Reply:
変更いたしました！
ご確認よろしくお願い致します。
1ee5c33cecfe23e0a194393b2ca5627e65f7116b

Reply:
@simamumu89 
ありがとうございます！
`Patients` だと`複数人の患者`を表すことにならないでしょうか？
クラス名/変数名は複数形の英単語になると、複数の要素を持ったList（配列）を表すように認識されてしまうことがあります。
1人の感じを表しますので、`Patient`にした方が適切かと思ったのですが、いかがでしょうか！？

Reply:
かしこまりました。
変更して対応致します。

Reply:
対応しました。
ご確認よろしくお願いします。
5007c72a26963442b9a19fcb8648bd8c535eb39f